readme.md

```diff
@@ -81,7 +86,7 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+pbj = Sandwich.create(type: "pbj", student_id: Student.find_by(name: "Geraldo").id)
```

Comment:
This is correct, but hard to read due to nested calls. I'd suggest breaking it up into two lines:

```ruby
geraldo = Student.find_by(name: "Geraldo")
pbj = Sandwich.create(type: "pbj", student_id: geraldo.id)
```